seacrowd/sea_datasets/wikimatrix/wikimatrix.py

```diff
@@ -0,0 +1,156 @@
+
+
+class WikiMatrix(datasets.GeneratorBasedBuilder):
+    """WikiMatrix is automatically extracted parallel sentences from the content of Wikipedia articles in 96 languages, including several dialects 
+    or low-resource languages."""
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"wikimatrix_{subset.replace('-', '_')}_source",
+            version=datasets.Version(_SOURCE_VERSION),
+            description="Bible En-Id source schema",
+            schema="source",
+            subset_id=f"wikimatrix_{subset.replace('-', '_')}",
+        ) 
+        for subset in _SUBSETS
+    ] + [
+        SEACrowdConfig(
+            name=f"wikimatrix_{subset.replace('-', '_')}_seacrowd_t2t",
+            version=datasets.Version(_SEACROWD_VERSION),
+            description="WikiMatrix Nusantara schema",
+            schema="seacrowd_t2t",
+            subset_id=f"wikimatrix_{subset.replace('-', '_')}",
+        )
+        for subset in _SUBSETS
+    ]
+
+    DEFAULT_CONFIG_NAME = "wikimatrix_en_id_source"
+
+    def _info(self):
+        if self.config.schema == "source":
+            features = datasets.Features({
+                "id": datasets.Value("string"), 
+                "text_1": datasets.Value("string"), 
+                "text_2": datasets.Value("string"),
+                "text_1_name": datasets.Value("string"), 
+                "text_2_name": datasets.Value("string"),
+            })
+        elif self.config.schema == "seacrowd_t2t":
+            features = schemas.text2text_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        lang1, lang2 = self.config.name.split("_")[1], self.config.name.split("_")[2]
+        filepath = Path(dl_manager.download_and_extract(_URLs[_DATASETNAME].replace("{lang1}", lang1).replace("{lang2}", lang2)))
```

Comment:
I don't mind this approach, but maybe you can use string formatting (`.format`) here? It seems more idiomatic for this use-case.